mutt_logging.c

```diff
@@ -68,45 +68,6 @@ static void error_pause(void)
   mutt_date_sleep_ms(sleep - elapsed);
 }
 
-/**
- * rotate_logs - Rotate a set of numbered files
- * @param file  Template filename
- * @param count Maximum number of files
- * @retval ptr Name of the 0'th file
- *
- * Given a template 'temp', rename files numbered 0 to (count-1).
- *
- * Rename:
- * - ...
- * - temp1 -> temp2
- * - temp0 -> temp1
- */
-static const char *rotate_logs(const char *file, int count)
-{
-  if (!file)
-    return NULL;
-
-  struct Buffer *old_file = mutt_buffer_pool_get();
-  struct Buffer *new_file = mutt_buffer_pool_get();
-
-  /* rotate the old debug logs */
-  for (count -= 2; count >= 0; count--)
-  {
-    mutt_buffer_printf(old_file, "%s%d", file, count);
-    mutt_buffer_printf(new_file, "%s%d", file, count + 1);
-
-    mutt_buffer_expand_path(old_file);
-    mutt_buffer_expand_path(new_file);
```

Comment:
I can't think of a reason why we're expanding these files each time.

Reply:
Hmm... anything to do with the `~` in the default `$debug_file`?

Reply:
Sure, but this doesn't change as we increment a counter to it